src/sentry/static/sentry/app/views/releases/detail/overview/chart/index.tsx

```diff
@@ -40,6 +43,61 @@ type Props = Omit<ReleaseStatsRequestRenderProps, 'crashFreeTimeBreakdown'> & {
 };
 
 class ReleaseChartContainer extends React.Component<Props> {
+  getTransactionsChartColors(): [string, string] {
+    const {yAxis} = this.props;
+
+    switch (yAxis) {
+      case YAxis.FAILED_TRANSACTIONS:
+        return [theme.red300, theme.red100];
+      default:
+        return [theme.purple300, theme.purple100];
+    }
+  }
+
+  getChartTitle() {
+    const {yAxis, organization} = this.props;
+
+    switch (yAxis) {
+      case YAxis.SESSIONS:
+        return {
+          title: t('Session Count'),
+          help: t('The number of sessions in a given period.'),
+        };
+      case YAxis.USERS:
+        return {
+          title: t('User Count'),
+          help: t('The number of users in a given period.'),
+        };
+      case YAxis.SESSION_DURATION:
+        return {title: t('Session Duration'), help: null};
```

Comment:
you don't need to set help to null. If you don't return help, it will be undefined :)